.github/workflows/test.yml

```diff
@@ -12,8 +12,8 @@ jobs:
       fail-fast: false
       matrix:
         version:
-         - 5.2.0
          - 5.4.0
+         - 6.0.0
```

Comment:
Sorry for commenting on a DRAFT PR :-), let's take this opportunity and also remove `2022.1` and `2022.2` releases, we should add `2024.1` instead

Reply:
`2024.1` - aleady there... 
`2022.1` and `2022.2` - we could, but basically we still have it in release status dashboard